src/pretalx/static/cfp/js/profile.js

```diff
@@ -0,0 +1,10 @@
+$('#id_get_gravatar').click(() => {
+    if ($('#id_get_gravatar').is(':checked')) {
+        $('.user-avatar-display').slideUp();
```

Comment:
When a speaker checks this box the file upload vanishes, but nothing else happens. It would be nice to load the gravatar picture, so the speaker knows, if the box actually did something.

Reply:
Done.